packages/cli/lib/project-generator.js

```diff
@@ -226,6 +226,7 @@ module.exports = class ProjectGenerator extends BaseGenerator {
       this.destinationPath(''),
       {
         project: this.projectInfo,
+        cliVersion,
```

Comment:
Is `cliVersion` used in a template?

Reply:
Yes, I am putting a message similar to what Angular says when you create a project with "this has been generated with Loopback CLI version X"

Reply:
Please note we already captures cli version in `.yo-rc.json`.

Reply:
I know, this that I am putting in the README file only has a visual and informative character. It has no other functionality

Reply:
@frbuceta , thank you for this PR. It is a good idea to add more content to this readme.

Reply:
@frbuceta , I will commit some changes today based on my suggestions...to help you land this PR.